### ISSUE TITLE
ошибка при анализе устава

### ISSUE THREAD

**Issue by SergeyShalaevsky** [nemoware/analyser#224]
на версии 2.0.1

Traceback (most recent call last):
  File "C:\work\analyser\analyser\runner.py", line 92, in process
    self.parser.find_attributes(legal_doc, context)
  File "C:\work\analyser\analyser\charter_parser.py", line 224, in find_attributes
    self._ebmedd(_charter)
  File "C:\work\analyser\analyser\charter_parser.py", line 200, in _ebmedd
    doc.sentences_embeddings = embedd_sentences(doc.sentence_map, self.elmo_embedder_default)
  File "C:\work\analyser\analyser\legal_docs.py", line 554, in embedd_sentences
    if len(text_map) > max_tokens:
TypeError: object of type 'NoneType' has no len()

**Comment by compartia** [comment]
теперь на веб-сервере
```
TypeError: Cannot read property 'subsidiary' of null
    at exports.getDocument (/Users/artem/work/nemo/goil/gpn-ui/projects/server/controller/document-controller.js:117:17)
    at process._tickCallback (internal/process/next_tick.js:68:7)```

**Comment by compartia** [comment]
fixed with Release v2.0.4
